qiskit_ibm_provider/ibm_backend_service.py

```diff
@@ -215,13 +227,23 @@ def jobs(
         # Build the filter for the query.
 
         api_filter = {}  # type: Dict[str, Any]
+        if isinstance(status, JobStatus):
+            status = status.name
         if isinstance(status, list):
+            status = [x.name if isinstance(x, JobStatus) else x for x in status]
```

Comment:
nit - it'd be nice to convert `status` to upper case so users won't have to remember if they're supposed to specify upper or lower case. Especially since right now it's a mixture (i.e. we have "pending" and "RUNNING").